src/test/resources/features/SearchDatasetByDescription.feature

```diff
@@ -0,0 +1,18 @@
+Feature: SearchDatasetByDescription
+  In order to list all datasets that match with a given description words
+  As a data user
+  I want to show the datasets that match with a description
+
+  Scenario: Show a dataset given a description
+    Given There is a dataset with description "dataset owned by owner" and owner "owner"
+    And There is 1 datasets registered
+    And I login as "owner" with password "password"
```

Comment:
Please, add more commits to this branch to address the previous comment (commit and then push). No need to reopen the Pull Request. It will be updated with new commits to your branch automatically